corehq/apps/userreports/reports/specs.py

```diff
@@ -484,6 +485,32 @@ def get_fields(self, data_source_config=None, lang=None):
         return self.numerator.get_fields() + self.denominator.get_fields()
 
 
+class ArrayAggLastValueReportColumn(ReportColumn):
+    type = TypeProperty('array_agg_last_value')
+    field = StringProperty(required=True)
+    order_by_col = StringProperty(required=False)
+    _agg_column_type = ArrayAggColumn
```

Comment:
why make this a class property?

Reply:
just took hint from other classes declared which do the same thing.
My guess is a class property not meant to be modified. Also, Keeping it as class property makes it clear what aggregation is happening within it.